File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/applikasjon/domenemodeller/KvartalsvisSykefraværshistorikk.java
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.domenemodeller;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

import java.util.List;

@Data
public class KvartalsvisSykefraværshistorikk {
  private Statistikkategori type;
  private String label;

  @JsonProperty("kvartalsvisSykefraværsprosent")
  private List<SykefraværForEttKvartal> sykefraværForEttKvartal;
}

File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/applikasjon/domenemodeller/KvartalsvisSykefraværshistorikk.kt
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.domenemodeller

import com.fasterxml.jackson.annotation.JsonProperty

data class KvartalsvisSykefraværshistorikk(
    val type: Statistikkategori? = null,
    val label: String? = null,
    @JsonProperty("kvartalsvisSykefraværsprosent")
    val sykefraværForEttKvartal: List<SykefraværForEttKvartal>? = null
)
